apitools/base/protorpclite/messages.py

```diff
@@ -250,8 +251,7 @@ class MyMessage(Message):
         outer_definition_name = cls.outer_definition_name()
         if outer_definition_name is None:
             return six.text_type(cls.__name__)
-        else:
-            return u'%s.%s' % (outer_definition_name, cls.__name__)
+        return u'%s.%s' % (outer_definition_name, cls.__name__)
```

Comment:
Regarding coverage. It is a fraction of # of lines covered / # total lines.
If you subtract 1 from the numerator and denumerator the value of the fraction will go down. If it went up that would mean we removed some previously non-covered code, so it would be more reason for concern. So coverage going up is not always good thing.

All the changes in this PR should be noop.